src/Microsoft.Docs.LearnValidation/Models/ValidatorHierarchyItem.cs

```diff
@@ -29,5 +31,29 @@ public class ValidatorHierarchyItem : HierarchyItem, IValidateModel
 
         [JsonProperty("page_kind")]
         public string PageKind { get; set; } = "";
+
+        [JsonProperty("depotName")]
```

Comment:
We typically don't put a `JsonProperty` for _camelCase_ properties since that's the default behavior in this project. We only add `JsonProperty` that does not adhere to this convention. 


~~[JsonProperty("depotName")]~~